### PR TITLE
Chore/Nessie run system tests on Google Storage

### PR DESCRIPTION
**Pull request by nopcoder**

### CODE REVIEW

File: .github/workflows/nessie.yaml [hunk 1]
```diff
@@ -45,3 +45,19 @@ jobs:
           AWS_SECRET_ACCESS_KEY: ${{ secrets.NESSIE_AWS_SECRET_ACCESS_KEY }}
         run: |
             docker-compose ps -q postgres && docker-compose exec postgres pg_dumpall --username=lakefs | gzip | aws s3 cp - s3://nessie-system-testing/${{ github.run_number }}/dump.gz
+      - name: Run lakeFS GS
```
Comment (itaiad200) [inline .github/workflows/nessie.yaml:48]: Don't you prefer to run this in parallel? There's no need to wait for S3 to finish.
Also, It won't run if S3 failed. Might have some added value to know if both failed, or just S3.

File: api/api_controller.go [hunk 2]
```diff
@@ -544,6 +544,10 @@ func (c *Controller) CreateRepositoryHandler() repositories.CreateRepositoryHand
 
 		err = ensureStorageNamespaceRW(deps.BlockAdapter, swag.StringValue(params.Repository.StorageNamespace))
 		if err != nil {
+			c.deps.logger.
+				WithError(err).
+				WithField("storage_namespace", swag.StringValue(params.Repository.StorageNamespace)).
+				Warn("Cloud not access storage namespace")
```
Comment (itaiad200) [inline api/api_controller.go:550]: Typo: "Could".

File: block/factory/build.go [hunk 3]
```diff
@@ -91,7 +91,7 @@ func buildGSAdapter(params params.GS) (*gs.Adapter, error) {
 		opts = append(opts, option.WithCredentialsFile(params.CredentialsFile))
 	} else if params.CredentialsJSON != "" {
 		ctx := context.Background()
-		cred, err := google.CredentialsFromJSON(ctx, []byte(params.CredentialsJSON))
+		cred, err := google.CredentialsFromJSON(ctx, []byte(params.CredentialsJSON), "https://www.googleapis.com/auth/cloud-platform")
```
Comment (itaiad200) [inline block/factory/build.go:94]: Please use const and document this address (maybe it's already a const in google's pkg that you can reuse).

File: docker-compose.yaml [hunk 4]
```diff
@@ -7,13 +7,15 @@ services:
...
-      LAKEFS_BLOCKSTORE_S3_CREDENTIALS_ACCESS_KEY_ID: ${AWS_ACCESS_KEY_ID}
-      LAKEFS_BLOCKSTORE_S3_CREDENTIALS_ACCESS_SECRET_KEY: ${AWS_SECRET_ACCESS_KEY}
+      - LAKEFS_AUTH_ENCRYPT_SECRET_KEY=some random secret string
+      - LAKEFS_DATABASE_CONNECTION_STRING=postgres://lakefs:lakefs@postgres/postgres?sslmode=disable
+      - LAKEFS_BLOCKSTORE_TYPE=${LAKEFS_BLOCKSTORE_TYPE}
+      - LAKEFS_BLOCKSTORE_LOCAL_PATH=/home/lakefs
+      - LAKEFS_GATEWAYS_S3_DOMAIN_NAME=s3.local.lakefs.io:8000
+      - LAKEFS_BLOCKSTORE_S3_CREDENTIALS_ACCESS_KEY_ID=${AWS_ACCESS_KEY_ID}
+      - LAKEFS_BLOCKSTORE_S3_CREDENTIALS_ACCESS_SECRET_KEY=${AWS_SECRET_ACCESS_KEY}
+      - LAKEFS_LOGGING_LEVEL=DEBUG
```
Comment (itaiad200) [inline docker-compose.yaml:17]: Remove.

File: docker-compose.yaml [hunk 5]
```diff
@@ -7,13 +7,15 @@ services:
...
+      - LAKEFS_BLOCKSTORE_GS_CREDENTIALS_JSON
```
Comment (itaiad200) [inline docker-compose.yaml:18]: Do we want this in our general `docker-compose` file?

**Review by itaiad200 — CHANGES_REQUESTED** [review]
Parallelising different Nessie runs.

File: .github/workflows/nessie.yaml (hunk 1 above)
Comment (nopcoder) [reply .github/workflows/nessie.yaml:48]: can't run steps in parallel - will open task for passing docker image build as it the step that takes the longest.
running the GS test takes less than half a min so I'm fine with that for now.

File: docker-compose.yaml (hunk 5 above)
Comment (nopcoder) [reply docker-compose.yaml:18]: yes, unless you think this is the point we should split? - it should still work

File: docker-compose.yaml (hunk 4 above)
Comment (nopcoder) [reply docker-compose.yaml:17]: I'll keep the env for control and remove the value

**Review by itaiad200 — APPROVED** [review]

**Pull request merged**